samples/subsys/mgmt/mcumgr/smp_svr/boards/tlsr9518adk80d.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022 Telink Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,console = &uart1;
+	};
+};
+
+&uart0 {
+	status = "okay";
+	current-speed = <115200>;
+	pinctrl-0 = <&uart0_tx_pb2_default &uart0_rx_pb3_default>;
+	pinctrl-names = "default";
+};
```

Comment:
Why are these being added to the smp_svr sample? The main UART should be part of the board dts not a specific application overlay

Reply:
Hi @lairdjm 

Thanks for remark. I've been removed uart0 from overlay but uart1 is still there, due to it is not part of dts for tlsr9518adk80d board. Shall I move uart1 to tlsr9518adk80d board as well?

Pretty the same approach as I've been used I saw here: https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts

Regards,
Alex

Reply:
It needs to be part of the board DTS yes, you cannot add an overlay for this board to every single sample application, that is untenable and a major maintenance burden

Reply:
@lairdjm 
Ok, got it. The .overlay currently removed. I guess dts stuff would be maintained by separate PR.